exampleCourse/questions/demo/annotated/engLogic/question.html

```diff
@@ -9,28 +9,34 @@
 <pl-hide-in-panel answer="true">
 <pl-overlay width="500" clip="false">
     <pl-background>
-      <pl-figure file-name="logic-diagram.png" directory="clientFilesQuestion"></pl-figure>
+      <div id="logic-diagram">
+        <pl-figure
+        file-name="logic-diagram.png"
+        alt="A three-input logic diagram. The top line, x, feeds through an arrowhead and into the final gate. The middle line, y, feeds through a triangle with a circle on the right, then a semicircle. The bottom line, z, feeds to the semicircle. The semicircle feeds into the final gate."
+        directory="clientFilesQuestion"
+        ></pl-figure>
+      </div>
     </pl-background>
       <pl-location left="60" top="15">
-        <pl-dropdown sort="ascend" answers-name="x">
+        <pl-dropdown sort="ascend" answers-name="x" aria-labelledby="logic-diagram">
```

Comment:
Labelling an input by a div containing an image seems strange to me.

Reply:
For reference:
![CleanShot 2025-04-24 at 14 47 02@2x](https://github.com/user-attachments/assets/8995b344-6e8d-42b7-816f-11a552d1aafa)

Reply:
I think that's the way to go. Labelling an input by the image as a whole doesn't seem useful to someone using a screen reader.

If you haven't already, fire up VoiceOver/NVDA and try interacting with these questions. We need to think beyond just "make the automated checker happy" 🙂

Reply:
Changed in [785357d](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/785357da395b470176b257d2ad9cac4482f141ef).